lemminx-maven/src/main/java/org/eclipse/lemminx/maven/MavenCompletionParticipant.java

```diff
@@ -248,6 +250,8 @@ public void onXMLContent(ICompletionRequest request, ICompletionResponse respons
 		case "configuration":
 			MavenPluginUtils.collectPluginConfigurationParameters(request, cache, repoSession, pluginManager).stream()
 					.map(parameter -> toTag(parameter.getName(), MavenPluginUtils.getMarkupDescription(parameter), request))
+					.filter(distinctByKey(
```

Comment:
This filter logic should probably be moved to `MavenPluginUtils.collectPluginConfigurationParameters` to prevent other future duplicate configuration parameters from popping up elsewhere.

Reply:
The duplicates actually aren't occurring in MavenPluginUtils.collectPluginConfigurationParameters.